install/connections.csv

```diff
@@ -86,48 +86,48 @@ CARD,10,Cairn Hill,Portable,,2010-09-04T23:23:02Z,2010-09-29T00:00:00Z
 CASA,10,Glenthorne,Short Period,,1994-06-19T20:22:00Z,1994-08-06T11:15:00Z
 CASM,10,Castle River,Short Period,,1993-10-18T05:51:00Z,1994-03-11T20:39:00Z
 CAW,10,Cannon Point,,,2003-05-20T02:00:00Z,9999-01-01T00:00:00Z
-CAW,11,Cannon Point,,,2003-05-20T02:00:00Z,2003-05-20T03:00:00Z
+CAW,11,Victoria University,EARSS Network,,2003-05-20T02:00:00Z,2003-05-20T03:00:00Z
 CCKO,10,Cattle Creek,Portable,,2014-10-11T02:01:00Z,2015-04-11T21:00:00Z
 CCW,10,Cape Campbell,,,2003-06-15T04:50:00Z,2003-06-19T13:30:00Z
-CCW,11,Cape Campbell,,,2003-06-15T04:50:00Z,2004-04-13T20:00:00Z
+CCW,11,Victoria University,EARSS Network,,2003-06-15T04:50:00Z,2004-04-13T20:00:00Z
```

Comment:
similar to `CAW` example, should these entries be something like
`CCW,10,Cape Campbell,EARSS Network,,2003-06-15T04:50:00Z,2003-06-19T13:30:00Z`
`CCW,11,Cape Campbell,EARSS Network,,2003-06-15T04:50:00Z,2004-04-13T20:00:00Z`

?

Reply:
Same problem, the analogue signal was digitised at Vic,, where as the digital signal locally.

The site would have again changes from 10 -> 11 (old kit), and new sensor (perhaps) renamed to 10 locally.